### DISCUSSION TITLE
getting ns0 tag in MetaDataBlock when saving as new file

### DISCUSSION THREAD

**Discussion by bladeyfocal** [danielhrisca/asammdf#505] [Q&A]
Hi everybody, i discovered an issue for me when reading an mf4 File and just save this one under a new file name. I'm performing this on a MacBook

Like so:

`from asammdf import MDF`
`mdf = MDF('/Users/myUser/somePath/openFile.mf4')`
`mdf.save(dst='/Users/myUser/someOtherPath/newFile.mf4', overwrite=True)`

if i open this new file on a windows machine with the MDF Validator, HeaderBlock seems to has an Issue with the MetaDataBlock

Message is "XML does not contain root tag <HDcomment>

![image](https://user-images.githubusercontent.com/49938227/114574340-74d4a180-9c79-11eb-91bc-1a96d3af4376.png)

in comparison to the original file, when i have a look in the md_data, each tag is getting an ns0: tag. 
e.g. `<ns0:HDcomment xmlns:ns0="http://www.asam.net/mdf/v4"><ns0:TX>`

any ideas whats going wrong here?

Tried to find a solution in the docs, but found nothing. 

goal of my project will be to read MetaData of the mf4 File, validate this and give them some changes if necessary. 

Is that possible to realize?

Best Regards

**Comment by danielhrisca** [comment]
Can you share the initial comment and the saved comment (just edit the textual data for your privacy but keep the xml tags and structure)?
